### PR TITLE
Krona: use better plot names

### PR DESCRIPTION
**Pull request by shiltemann**
use the dataset names as IDs instead of `0,1,2,..` when making multisample krona plots

ping @bebatut

### CODE REVIEW

File: tools/taxonomy_krona_chart/taxonomy_krona_chart.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
         -o '${output}'
         ${combine_inputs}
         #for $input_count, $input_dataset in enumerate( $type_of_data.input ):
-            '${input_dataset},${input_count}'
+            '${input_dataset},${input_dataset.element_identifier}'
```
[the remark this answers is not in the document]
Comment (bgruening) [reply tools/taxonomy_krona_chart/taxonomy_krona_chart.xml]: @mvdbeek do you think we should have something like `element_identifier_save`?

[the remark this answers is not in the document]
Comment (shiltemann) [reply tools/taxonomy_krona_chart/taxonomy_krona_chart.xml]: thanks Marius, done :)

[the remark this answers is not in the document]
Comment (mblue9) [reply tools/taxonomy_krona_chart/taxonomy_krona_chart.xml]: >do you think we should have something like element_identifier_save

@bgruening do you mean have a shortcut for adding the escaped element_identifier? If so, then yes please!

**Comment by mblue9** [comment]
(btw I like this "Tool Name: what the PR is for" as a PR title system. I'm going to adopt that now as that's one less character than what I had started doing "Tool Name - " 😄)

File: tools/taxonomy_krona_chart/taxonomy_krona_chart.xml [hunk 2]
```diff
@@ -20,7 +22,8 @@
         -o '${output}'
         ${combine_inputs}
         #for $input_count, $input_dataset in enumerate( $type_of_data.input ):
```
Comment (bebatut) [inline tools/taxonomy_krona_chart/taxonomy_krona_chart.xml]: We probably do not need an enumerate if we do not use the count

**Pull request merged**